test/data/docker-states/simple/inspect.json

```diff
@@ -34,9 +34,18 @@
 	"AppArmorProfile": "",
 	"ExecIDs": null,
 	"HostConfig": {
-		"Binds": [
-			"/tmp/balena-supervisor/services/1011165/main:/tmp/resin",
-			"/tmp/balena-supervisor/services/1011165/main:/tmp/balena"
+		"Binds": null,
+		"Mounts": [
+			{
+				"Type": "bind",
+				"Source": "/tmp/balena-supervisor/services/1011165/main",
+				"Target": "/tmp/resin"
+			},
+			{
+				"Type": "bind",
+				"Source": "/tmp/balena-supervisor/services/1011165/main",
+				"Target": "/tmp/balena"
+			}
```

Comment:
Changed from `Binds` to `Mounts` in these 3 test JSON files to ensure the `Creating service instances from docker configuration` ([ref](https://github.com/balena-os/balena-supervisor/blob/3a315db354c2c937b2a3a8d37d3fe00196b6a192/test/src/compose/service.spec.ts#L1015-L1046)) test suites passed.

Reply:
On a device, when inspecting a container that's created with the corrrect HostConfig.Mounts config, this is what you would expect to see, so changing the inspect.json data in this way is in line with how the change should be reflected on devices.

Reply:
Reverted the JSON changes since we're using Binds instead of Mounts for update lock bind mounts.